src/Is/fqdn.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @since 0.0.2
+ */
+
+import { isString } from "./string";
+
+/**
+ * check if it's Fully qualified domain name
+ *
+ * @since 0.0.2
+ * @category Is
+ */
+export const isFqdn = <A>(val: A) => {
+  if (!val || !isString(val)) {
+    return false;
+  }
+  const _val: string = "" + val;
+  const parts = _val.split(".");
+  const tld = parts.pop() ?? "";
```

Comment:
`pop` method is mutating the object, I think it's better to use `parts.slice(-1)` instead
IMHO, we need to sanitize the value first because the TLD might not a valid if the value doesn't comply with fqdn format. for instance, if the value is `www.badexample.com/accessToken=eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiJlNjc4ZjIzMzQ3ZTM0MTBkYjdlNjg3Njc4MjNiMmQ3MCIsImlhdCI6MTQ2NjYzMzMxNywibmJmIjoxNDY2NjMzMzE3LCJleHAiOjE0NjY2MzY5MTd9.rgx_o8VQGuDa2AqCHSgVOD5G68Ld_YYM7N7THmvLIKc` then tld value would be `rgx_o8VQGuDa2AqCHSgVOD5G68Ld_YYM7N7THmvLIKc` instead of `com`